airflow/utils/helpers.py

```diff
@@ -31,10 +31,17 @@
 from airflow.exceptions import AirflowException
 from airflow.utils.module_loading import import_string
 
+if TYPE_CHECKING:
+    from airflow.models import TaskInstance
+
 KEY_REGEX = re.compile(r'^[\w.-]+$')
+CAMELCASE_TO_SNAKE_CASE_REGEX = re.compile('(?!^)([A-Z]+)')
```

Comment:
I’m surprised for all the opinions pylint have, it does not bother to enforce a regex pattern must always use an r-string.